src/test/java/com/wikia/webdriver/testcases/globalshortcuts/KeyboardShortcutsTest.java

```diff
@@ -9,7 +9,7 @@
 import com.wikia.webdriver.common.templates.NewTestTemplate;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.HomePage;
 
-@Execute(asUser = User.USER, onWikia = "globalshortcuts-en")
```

Comment:
Good point, @SebastianMarzjan! :) We've got [tests for interstitial](https://github.com/Wikia/selenium-tests/blob/master/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsInterstitial.java#L42) the most important cases for us are tested it. However, we won't mind if somebody adds to it test checking if the close button works :japanese_ogre: